modules/creating-a-project-workbench.adoc

```diff
@@ -4,15 +4,15 @@
 = Creating a project workbench
 
 [role='_abstract']
-To examine and work with data models in an isolated area, you can create a workbench. This workbench enables you to create a new Jupyter notebook from an existing notebook container image to access its resources and properties. For data science projects that require data to be retained, you can add container storage to the workbench you are creating.
+To examine and work with data models in an isolated area, you can create a workbench. This workbench enables you to create a Jupyter notebook from an existing notebook container image to access its resources and properties. For data science projects that require data retention, you can add container storage to the workbench you are creating. If you require extra power for use with large datasets, you can assign accelerators to your workbench to optimize performance. 
```

Comment:
We could avoid 'enables you' here by adjusting slightly to something like, 'You can use this workbench to...'

Reply:
Changed.